src/Service/ProgrammesAggregationService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Service;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ClipRepository;
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+use BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\CoreEntityMapper;
+use BBC\ProgrammesPagesService\Cache\CacheInterface;
+
+class ProgrammesAggregationService extends AbstractService
+{
+    public function __construct(
+        CoreEntityRepository $repository,
+        CoreEntityMapper $mapper,
+        CacheInterface $cache
+    ) {
+        parent::__construct($repository, $mapper, $cache);
+    }
+
+    public function findChildrensByType(
+        array $findByAncestryIds,
+        string $type,
+        ?int $limit = self::DEFAULT_LIMIT,
+        int $page = self::DEFAULT_PAGE
+    ) {
+        return $this->repository->findProgrammesByAncestryAndType(
```

Comment:
`ProgrammeAggregationsService` not, `ProgrammesAggregationService`. The whole thing is a plural.